plugins/pipette/src/pipette.cc

```diff
@@ -0,0 +1,223 @@
+#include "pipette.hh"
+
+#include <gdk/gdk.h>
+#include <cmath>
+
+#include <scroom/pipetteviewinterface.hh>
+#include <scroom/unused.hh>
+
+////////////////////////////////////////////////////////////////////////
+// Pipette
+////////////////////////////////////////////////////////////////////////
+
+Pipette::Pipette()
```

Comment:
Not a big fan of empty constructors/destructors. Can we delete them?

Reply:
Sure